cmd/determinize-ci-operator/main.go

```diff
@@ -34,6 +69,10 @@ func main() {
 			return nil
 		}
 
+		if sets.NewString(o.enabledTemplateMigrations.Strings()...).Has(openshiftInstallerSRCTemplateName) && migratedCount <= o.templateMigrationCeiling {
+			migratedCount += migrateOpenshiftInstallerSRCTemplates(&output, sets.NewString(o.templateMigrationAllowedBranches.Strings()...), sets.NewString(o.templateMigrationAllowedOrgs.Strings()...), sets.NewString(o.templateMigrationAllowedClusterProfiles.Strings()...))
```

Comment:
might make sense to upstream a `.Set()`

Reply:
yeah, already did that last week, updated the PR